README.md

```diff
@@ -7,11 +7,10 @@
 
 ## Quick Links
 
-* See [INSTALL.md](INSTALL.md) for building instructions.
 * [Try LiquidHaskell in your browser](http://goto.ucsd.edu:8090/index.html)
+* [Use LiquidHaskell in your stack/cabal project](https://ucsd-progsys.github.io/liquidhaskell/plugin/)
```

Comment:
This documentation doesn't explain about adding `liquid-base`, `liquid-containers`, etc, to package dependencies. And it only tells you to add `liquidhaskell` to dependencies if you care to read the section titled "Benefits"!

I don't oppose making these changes, but I think I'd like the README to contain the up-to-date information one day, without sending the user through links.

Reply:
How do you suggest we avoid staleness without using links?

This project already has 2 websites and 4+ tutorials, with a lot of duplicated and out of date information.
Adding non-link information to the README seems like it will aggravate this situation in the long run.

Reply:
Perhaps we should move in the other direction and cut out even more of the README links to make it really clear to newbies that they should go to [the docs website](https://ucsd-progsys.github.io/liquidhaskell/).

Reply:
Yes I agree with this - otherwise the README will again get too big.

Reply:
> Perhaps we should move in the other direction and cut out even more of the README links to make it really clear to newbies that they should go to the docs website.

This could work + improving the documentation in the website :+1: